app/src/main/java/com/codesoom/assignment/UserNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.codesoom.assignment;
+
+public class UserNotFoundException extends RuntimeException{
```

Comment:
포매터를 실행하는 습관을 들이거나 자동으로 포매터가 동작하도록 설정해 보세요.
```suggestion
public class UserNotFoundException extends RuntimeException {
```

Reply:
자동으로 정렬 하는 건 `Save Actions 자동정렬 플러그인` 사용하는 건가요 ??

Reply:
방법이 다양합니다. 말씀하신 Save Actions를 쓰는 것도 방법 중 하나입니다.

Reply:
넵! 이것저것 시도해보겠습니다!